config/locales/en/tasklist.yml

```diff
@@ -26,6 +26,11 @@ en:
       not_applicable: Not applicable
 
   crime_applications:
+    index:
+      page_title: Your applications
+      heading: Your applications
+      start_button: Make a new application
+      delete_buttons: Delete
```

Comment:
Not that it matters a lot but should this key be singular? 😄

Reply:
Ha ha Yep it should be will change!